tools/toolchain/image

```diff
@@ -1 +1 @@
-docker.io/scylladb/scylla-toolchain:fedora-40-20241219
+docker.io/scylladb/scylla-toolchain:fedora-40-20250224
```

Comment:
The rules say the toolchain has to include the branch name (https://github.com/scylladb/scylladb/tree/master/tools/toolchain#native-publishing-procedure-complicated) so different branches don't step on each other.

Reply:
I just renamed the old one

Reply:
I don't understand.

Reply:
you mentioned the name of the image. well the old one looks the same, also check older history

Reply:
When generating a toolchain for a branch, the image name has to include the branch name.

Reply:
ok, i will re-generate it, so it should be `scylla-toolchain:branch-2025.1-fedora-40-<date>` ?

Reply:
The doc gives as an example `scylladb/scylla-toolchain:fedora-29-[branch-3.0-]20181128`, so fedora-40-branch-2025.1-date

Reply:
changed in v2